app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<section class="index">
+  <ul>
+    <% @tasks.each do |todo_item| %>
+        <li>
+          Completion status: <%= todo_item.completed %><br/>
+          <%= link_to todo_item.task, task_path(todo_item.id) %>,
+          <%= button_to "Mark Complete", {action: "complete", id: todo_item.id}, method: :patch, class: "complete-button"  %>
+          <%= button_to "Mark Incomplete", {action: "complete", id: todo_item.id}, method: :patch, class: "incomplete-button"   %><br/>
```

Comment:
Both the "Mark Complete" and "Mark Incomplete" buttons go to the same controller action, which both toggle the completeness of the task. What is the effect? What happens when you manually test this feature out?